src/script.js

```diff
@@ -161,6 +176,7 @@ function isCanonicalSignature (buffer) {
 module.exports = {
   compile: compile,
   decompile: decompile,
+  toWitness: toWitness,
```

Comment:
>  I would name it decompilePushOnly, but the function changes some opcodes to Buffers.

I haven't got time at the moment, but could you reference to this behaviour in the BIPs?